project-base/tests/App/Test/Codeception/Module/StrictWebDriver.php

```diff
@@ -50,13 +51,13 @@ protected function findFields($selector)
         if (!is_array($selector) && !$selector instanceof WebDriverElement) {
             $message = 'Using findFields() with fuzzy locator is slow. '
                 . 'You should implement new method with strict locator. See ' . self::class;
-            throw new \Tests\App\Test\Codeception\Exception\DeprecatedMethodException($message);
+            throw new DeprecatedMethodException($message);
         }
         return parent::findFields($selector);
     }
 
     /**
-     * @deprecated
+     * @deprecated replaced by strict alternatives: clickByName() and clickByText()
```

Comment:
I suggest ignoring this file for the mandatory deprecated message in project-base. 

This file is barely updated and the necessity of fixing deprecated methods seems annoying